SuperBuild/External_CTK.cmake

```
@@ -71,7 +71,7 @@ if(NOT DEFINED CTK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "4c7154624e9a1a528ca04c067e7f35549339b53e"
+    "78c4ff2a0bad484ceb51ec35c2f239e1f4b0cf68"
```

This needs to be updated to the following per https://github.com/commontk/CTK/commit/e081be7dbbb5f3e3c4294c0edf9a0a96de815ee1:
```suggestion
    "e081be7dbbb5f3e3c4294c0edf9a0a96de815ee1"
```

---

Thanks for pointing this out. I thought you pushed my commit to master instead of "rebase and merge".